vllm/model_executor/layers/fused_moe/modular_kernel.py

```diff
@@ -141,6 +141,17 @@ def apply(self, output: Optional[torch.Tensor],
         raise NotImplementedError
 
 
+PrepareResultType = tuple[
```

Comment:
I think a `@dataclass` would be good here. If you prefer the tuple, let's just put comments on what each of the items represent.

Reply:
I'll add some comments.